### DISCUSSION TITLE
zephyr logging/trace

### DISCUSSION THREAD

**Discussion by DINESHKUMARK1** [thesofproject/sof#8397] [General]
Hi,
I am trying to understand the zephyr logging/trace ,where I can find the detailed information .

**Comment by lgirdwood** [comment]
@DINESHKUMARK1 I would check the Zephyr docs as SOF uses native zephyr logging. My understanding is that your platform will need a Zephyr logging backend if it does not support existing Zephyr logging backend drivers. Best to Intel platforms for examples that can be used as templates.

**Comment by DINESHKUMARK1** [comment]
ya ,thanks for the information ,will take a look at zephyr docs for logging .
